releasenotes/notes/qiskit-1.0-incompatibility-9ad22472837837d8.yaml

```
@@ -0,0 +1,26 @@
+---
+prelude: |
+    Qiskit 0.45.3 is a point release with no code changes other than to raise an :exc:`ImportError`
+    if it detects it has been installed in an invalid environment with Qiskit 1.0+.
+
+    The packaging structure of Qiskit is changing in Qiskit 1.0, and unfortunately the changed
+    requirements cannot be fully communicated to ``pip``, especially if ``pip install --upgrade``
+    commands are run after the environment has been initially configured.  All versions of Qiskit
+    prior to 1.0 (including this one) have an installation conflict with Qiskit 1.0 that ``pip``
+    will not resolve.
+
+    If ``import qiskit`` raises an :exc:`ImportError` for you, your environment is in an invalid
+    state, and versions of Qiskit 0.45/0.46 and 1.0 are both reachable, which will result in subtley
+    broken code.  You will need to create a new virtual environment, and ensure that *only* one of
+    the two versions are installed.  In particular, if you are intending to install Qiskit 1.0,
+    you must have no packages that depend on ``qiskit-terra`` installed; these packages are
+    incompatible with Qiskit 1.0 and must be updated.  If you are intending to install Qiskit 0.45
+    or 0.46, you must ensure that you have nothing attempting to install ``qiskit>=1.0``.
+
+    If you develop a library based on Qiskit and you still have a dependency on ``qiskit-terra``,
+    you should urgently release a new package that depends only on ``qiskit``.  Since version 0.44,
+    the ``qiskit`` package contained only the ``qiskit-terra`` compiler core (the component that is
+    now simply called "Qiskit").
```

I think maybe reinforcing that you can do this prior to qiskit 1.0 a bit more would be good. When I first read this I kind of got the impression that this was pushing for Qiskit 1.0 support. Maybe something like:

```suggestion
    now simply called "Qiskit"). As long as your library is compatible with qiskit/qiskit-terra >=0.44.0 you can
    substitute ``qiskit-terra`` with ``qiskit`` without issue (even if your library isn't ready for Qiskit 1.0 yet).
```

Although I'm not a huge fan of this suggestion either.

---

Added in 8ad7cf597d.